### PR TITLE
Define demo Grafana dashboards for operator and oauth service health and for measuring the outbound traffic.

### PR DESCRIPTION
**Pull request by metlos**
### What does this PR do?
This defines demo Grafana dashboards for operator and oauth service health and for measuring the outbound traffic.

### What issues does this PR fix or reference?
https://issues.redhat.com/browse/SVPI-182

obrazek
obrazek

### CODE REVIEW

**Review by sparkoo — APPROVED** [review]
is it automatically created with `deploy-minikube-monitoring.sh` ? does it make to update it ?

**Comment by metlos** [comment]
> I think it is not.
Thanks for the pointer. They should be auto-created now.

File: hack/grafana-dashboards/spi-health.json [hunk 1]
```diff
@@ -0,0 +1,618 @@
...
+  "templating": {
+    "list": []
+  },
+  "time": {
+    "from": "now-6h",
+    "to": "now"
+  },
+  "timepicker": {},
+  "timezone": "",
+  "title": "Health",
```
[the remark this answers is not in the document]
Comment (metlos) [reply hack/grafana-dashboards/spi-health.json]: These should be `SPI Health` and `SPI Outbound traffic` in the latest version.

File: hack/grafana-dashboards/spi-outbound-traffic.json [hunk 2]
```diff
@@ -21,6 +21,7 @@
       }
     ]
   },
+  "description": "The measurements of the outbound requests made by the operator.",
```
[the remark this answers is not in the document]
Comment (metlos) [reply hack/grafana-dashboards/spi-outbound-traffic.json:24]: It's in the name 🙂

**Pull request merged**